tutorials/text/annotations.py

```diff
@@ -388,6 +391,7 @@ def custom_box_style(x0, y0, width, height, mutation_size):
             arrowprops=dict(arrowstyle="simple",
                             connectionstyle="arc3,rad=-0.2"))
 
+# .. redirect-from:: /gallery/userdemo/annotate_simple03
```

Comment:
I think these all need to be at the top - only /gallery/userdemo/annotate_simple01 is working as written.  (See https://output.circle-artifacts.com/output/job/9942d81b-1653-44c6-bc69-93f51ab0a293/artifacts/0/doc/build/html//gallery/userdemo/annotate_simple03.html versus 
https://output.circle-artifacts.com/output/job/9942d81b-1653-44c6-bc69-93f51ab0a293/artifacts/0/doc/build/html//gallery/userdemo/annotate_simple01.html)